istioctl/pkg/multixds/gather.go

```diff
@@ -0,0 +1,93 @@
+// Copyright Istio Authors.
```

Comment:
If possible, I would move this from istioctl/... to pkg/adsc 

I don't think it's specific in any way to istioctl, and may be reused in other code ( including istiod itself )